src/keys/ecdh.js

```diff
@@ -1,41 +1,28 @@
 'use strict'
 
 const crypto = require('crypto')
-const nextTick = require('async/nextTick')
 
 const curves = {
   'P-256': 'prime256v1',
   'P-384': 'secp384r1',
   'P-521': 'secp521r1'
 }
 
-exports.generateEphmeralKeyPair = function (curve, callback) {
+exports.generateEphmeralKeyPair = async function (curve) { // eslint-disable-line require-await
```

Comment:
Does this need to be an async function?

Reply:
The browser implementation is async but the node implementation is sync.  I think the intention here is to present the same interface.  See @alanshaw's note titled 'async keyword' in the PR description.